.github/CODEOWNERS

```diff
@@ -1,3 +0,0 @@
-# These owners will be the default owners for everything in
-# the repo. Unless a later match takes precedence.
-* @gnosis/safe-services
```

Comment:
This should not be touched

Reply:
It's seems was due a merge conflict. It's solved.